biocrnpyler/components_basic.py

```diff
@@ -8,14 +8,41 @@
 from .reaction import Reaction
 from .species import Complex, Species
 
+class LengthComponent(Component):
+    """
+    A high level class for Components with Length
```

Comment:
So now all components will actually be "LengthComponent" objects? I'm not sure how I feel about that. Like in the test below, an RNA component is a "LengthComponent" object. This makes things confusing for the user who only works with the basic biocrnpyler classes. Is there a way around it so that every component is still a `Component` object?

Reply:
Why couldn't length just be an optional attribute of the `Component` class. Do we need this super class? I don't like the fact that DNA, RNA, etc are now "LengthComponent".

Reply:
I can't remember why I (or maybe someone else) did this - happy to remove LengthComponent.

Reply:
@ayush9pandey removed LengthComponent